### ISSUE TITLE
pepelsbey links in LICENSE

### ISSUE THREAD

**Issue by Guria** [vladkhvo/es2015-presintation#1]

**Comment by vkhv** [comment]
Is it problem? Should i remove it? Or it possible to change on my name? =)

**Comment by Guria** [comment]
just decided to inform you. seems you are copy-pasting this file without looking into it for some time.

**Comment by Guria** [comment]
I don't think that it's correct to have 3rd person name as Copyright note in LICENSE file. Not a big issue, of course. Decide yourself how to deal with it.

**Comment by vkhv** [comment]
Thank for your attention! What do you think if i just will change it to Copyleft. It that legal to use copyleft with MIT? =))
